source/installguide/configuration.rst

```diff
@@ -678,9 +678,8 @@ Advanced Zone Configuration
 
    -  **Protocol.** For XenServer, choose either NFS, iSCSI, or
       PreSetup. For KVM, choose NFS, SharedMountPoint, CLVM, and RBD.
-      For vSphere choose either VMFS (iSCSI or FiberChannel) or NFS. The
```

Comment:
any reason to not include (iSCSI or FiberChannel) for VMFS ?

Reply:
With the new vSphere advanced capabiltities, PreSetup can be of any storage type, so to keep it generic here we mentioned the storage technology names as VMFS, vSAN, vVOls